operator/src/main/resources/Operator.properties

```diff
@@ -215,6 +215,7 @@ WLSDO-0016=DomainUID ''{0}'' introspector job name ''{1}'' exceeds maximum allow
 WLSDO-0017=DomainUID ''{0}'' and cluster name ''{1}'' combination ''{2}'' exceeds maximum allowed length ''{3}''.
 WLSDO-0018=DomainUID ''{0}'' and server name ''{1}'' combination ''{2}'' exceeds maximum allowed length ''{3}''.
 WLSDO-0019=DomainUID ''{0}'' and admin server name ''{1}'' combination ''{2}'' exceeds maximum allowed length ''{3}''.
+WLSDO-0020="Domain ''{0}'' dynamic updated successfully. No restart is necessary"
```

Comment:
It's probably not necessary to pass in the domain name - as the logger automatically determines and includes the current domain name in most cases.  Also, a little word smithing would help here to provide more context.  Maybe something like "Model in Image online update to the model succeeded. No WebLogic Server pod restarts are necessary."

Reply:
Also, the text for any other new INFO/WARNING/SEVERE level logs should go here instead of being in-lined in the java.

Reply:
Should 'dynamic updated' by 'online updated'?